### PR TITLE
script/backport-create-issue: add debug messages for retry

### PR DESCRIPTION
**Pull request by batrick**
## Contribution Guidelines
- To sign and title your commits, please refer to [Submitting Patches to Ceph](https://github.com/ceph/ceph/blob/main/SubmittingPatches.rst).

- If you are submitting a fix for a stable branch (e.g. "quincy"), please refer to [Submitting Patches to Ceph - Backports](https://github.com/ceph/ceph/blob/master/SubmittingPatches-backports.rst) for the proper workflow.

- When filling out the below checklist, you may click boxes directly in the GitHub web UI.  When entering or editing the entire PR message in the GitHub web UI editor, you may also select a checklist item by adding an `x` between the brackets: `[x]`.  Spaces and capitalization matter when checking off items this way.

## Checklist
- Tracker (select at least one)
  - [ ] References tracker ticket
  - [ ] Very recent bug; references commit where it was introduced
  - [ ] New feature (ticket optional)
  - [ ] Doc update (no ticket needed)
  - [x] Code cleanup (no ticket needed)
- Component impact
  - [ ] Affects [Dashboard](https://tracker.ceph.com/projects/dashboard/issues/new), opened tracker ticket
  - [ ] Affects [Orchestrator](https://tracker.ceph.com/projects/orchestrator/issues/new), opened tracker ticket
  - [x] No impact that needs to be tracked
- Documentation (select at least one)
  - [ ] Updates relevant documentation
  - [x] No doc update is appropriate
- Tests (select at least one)
  - [ ] Includes [unit test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/tests-unit-tests/)
  - [ ] Includes [integration test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/testing_integration_tests/)
  - [ ] Includes bug reproducer
  - [x] No tests

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test classic perf`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test dashboard cephadm`
- `jenkins test api`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`
- `jenkins test windows`
- `jenkins test rook e2e`
</details>

### CODE REVIEW

File: src/script/backport-create-issue [hunk 1]
```diff
@@ -257,6 +257,8 @@ def update_relations(r, issue, dry_run):
             other = r.issue.create(**create_args)
         except ValidationError as e:
             # try without an assignee
+            logging.debug("Creating the issue failed: {e}")
```
Comment (idryomov) [inline src/script/backport-create-issue:260]: Nit: This is inside of a loop, so I'd make more precise:
```suggestion
            logging.debug("Creating issue for {release} failed: {e}")
```

**Review by idryomov — APPROVED** [review]

**Comment by idryomov** [comment]
`make check` failure is not related, but I'm not retriggering in case you decide to address the nit:
```
CMake Error at /usr/lib/x86_64-linux-gnu/cmake/protobuf/protobuf-targets.cmake:111 (message):
  The imported target "protobuf::protoc" references the file

     "/usr/bin/protoc-25.1.0"

  but this file does not exist.  Possible reasons include:

  * The file was deleted, renamed, or moved to another location.

  * An install or uninstall procedure did not complete successfully.

  * The installation package was faulty and contained

     "/usr/lib/x86_64-linux-gnu/cmake/protobuf/protobuf-targets.cmake"

  but not all the files it references.

Call Stack (most recent call first):
  /usr/lib/x86_64-linux-gnu/cmake/protobuf/protobuf-config.cmake:14 (include)
  src/CMakeLists.txt:384 (_find_package)
  src/seastar/cmake/SeastarDependencies.cmake:162 (find_package)
  src/seastar/CMakeLists.txt:399 (seastar_find_dependencies)
```
https://jenkins.ceph.com/job/ceph-pull-requests/139713/consoleText

File: src/script/backport-create-issue (hunk 1 above)
Comment (batrick) [reply src/script/backport-create-issue]: ack, look again

Comment (idryomov) [reply src/script/backport-create-issue]: Looks much better to me!

**Comment by batrick** [comment]
jenkins test api

**Pull request merged**